### DISCUSSION TITLE
Cadenas IV tp7

### DISCUSSION THREAD

**Discussion by Nikzza** [INGCOM-UNRN-P1/cursada-2023#188] [Preguntas y respuestas] [answered]
Supongo que es una cadena de caracteres, el ejercicio esta mal escrito.

![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141664052/488c9ae5-8e2a-4fdc-8228-2b98aee91d2e)

seria: 
char* elimina_duplicados(char* cadena, int tamano) 

no?

**Comment by martinvilu** [comment, marked as answer]
Se puede hacer con caracteres y con números.
```
char* elimina_duplicados(char* cadena, int* tamano)
```
Pero el tamaño cambia, y el retorno esta ocupado con 'otro' valor.

**Reply by Nikzza** [reply]
Hace falta usar un realloc para disminuir la memoria?

**Reply by martinvilu** [reply]
o una cadena nueva

**Reply by Nikzza** [reply]
joya

**Comment by martinvilu** [comment]
Mejor que cerrar el hilo, marcá la respuesta que responde la pregunta
